warehouse/accounts/models.py

```diff
@@ -95,6 +96,18 @@ class User(SitemapMixin, db.Model):
         "Macaroon", backref="user", cascade="all, delete-orphan", lazy=False
     )
 
+    events = orm.relationship(
+        "UserEvent", backref="user", cascade="all, delete-orphan", lazy=False
```

Comment:
I think this is _probably_ the right relationship, but I'm not sure about the cascade. It seems like we might want to keep these for some period post deletion? For instance in the event of an attacker taking control of an account and purging all projects then deleting the account.

Reply:
Good point! It looks like the default `cascade` is `"save-update, merge"`, so it looks like that's what we want IIUC.